src/NodeServices/PythonServices.cs

```diff
@@ -175,6 +178,18 @@ private PythonEngineManager()
             AppDomain.CurrentDomain.AssemblyLoad += new AssemblyLoadEventHandler((object sender, AssemblyLoadEventArgs args) => LoadDefaultPythonEngine(args.LoadedAssembly));
         }
 
+        internal static Tuple<string, string> TryGetPythonEnginePackage(string engineName)
+        {
+            if (Instance.PythonEnginePackageMap.Count > 0)
+            {
+                if (Instance.PythonEnginePackageMap.TryGetValue(engineName, out var tuple))
```

Comment:
what happens when there are engines with the same name?

Reply:
will add a check